packages/serverpod/lib/src/cache/local_cache.dart

```diff
@@ -85,14 +85,13 @@ class LocalCache extends Cache {
   ]) async {
     var entry = _entries[key];
 
-    if (entry != null &&
-        (entry.expirationTime?.compareTo(DateTime.now()) ?? 0) < 0) {
-      await invalidateKey(key);
-      return null;
-    }
-
     if (entry != null) {
-      return serializationManager.decode<T>(entry.serializedObject);
+      if (entry.isExpired) {
+        // invalidate key and generate a new value below
```

Comment:
Suggestion: Remove the comment, generally think comments should be avoided within the code. Very often these go out of sync with the code when it is changed.

Prefer refactoring the flow if it isn't obvious what is going on.

Reply:
Removed the "obvious" comments.